docu/Concepts/Snippets/1-Blockchain/Connector/Dezentralität.md

```diff
@@ -0,0 +1,70 @@
+## Dezentralität
+
+Ich habe von Anfang an versucht Gradido so dezentral wie möglich zu entwerfen.
+Dezentral an der Basis. Kryptografie und Blockchain-Technologie sind meiner Meinung nach die besten Technologien dafür.
+Sie ermöglichen einen Ansatz bei dem der Benutzer und seine Daten im Mittelpunkt stehen. In der Form,
+das nur der Benutzer selbst die volle Kontrolle über seine Daten haben kann. 
+Dadurch wird der Benutzer unabhägig von Server-Admins und das Gesamtsystem hat weniger Points of Failure.
```

Comment:
This is not how you implemented it. The login server holds all private keys of all users and the concept of handing out the private keys in form of word list failed to my knowledge. Even when the passphrase is know to the user, the server still centrally controls the private keys and the security of those are down to a symmetric password of users choice (with some password rules)

Reply:
> Sie **ermöglichen** einen Ansatz bei dem der Benutzer und seine Daten im Mittelpunkt stehen.
- 1. Die Blockchain Architektur ermöglicht das.
- 2. Ich kann nicht zaubern und alles auf einmal implementieren.
- 3. Der Plan war das wir mehr Mitarbeiter einstellen wenn mehr Geld da ist die mir helfen das alles umzusetzen!!!
Stattdessen wurde einfach erstmal alles nochmal programmiert mit einer angeblich schneller zu lernenden Programmiersprache in etwa derselben Zeit die ich gebraucht habe, aber mit mehr Menschen. 
Diese Verschwendung macht mich so wütend!!!

- 4. Argon2!!!!! Bruteforce: 40 H/s mit einem überdurchschnittlichen Gaming-PC, mindestens 8 Zeichen, mehr als 62 Möglichkeiten pro Zeichen: 62^8 / 40 / 60 / 60 / 24 / 356 = 177.463,8 Jahre
- 5. Wichtig ist das die Möglichkeit für Benutzer besteht, welche das Technische Verständnis dafür haben. Und für die Benutzer die es nicht haben, muss es einen einfacheren Weg geben. 
Unsere aktuelle (Test) Zielgruppe hat kein gutes Technisches Verständis, also musste ich erstmal die Alternativ-Version implementieren. Aber auch hier gibt es noch einige Möglichkeiten die Passphrasen abzusichern. Zum Beispiel das die Passphrase eines guten Freundes mit dem eigenen Verschlüsselungskey verschlüsselt wird. Oder das mehrere Keys hintereinander verwendet werden, so das nur mehrere gemeinsam das widerherstellen können. 
Oder das sie halt mit dem Key einer Vertauensperson der Community verschlüsselt ist, für die die niemanden kennen.
Aber all diese Möglichkeiten zu implementieren habe ich noch nicht geschafft. 
Oder das der Key bei einer Handy-App im Secure-Storage des Handys gespeichert wird. Dann sollte bloß das Handy nicht verloren gehen. Oder das man das auf mehreren Geräten sichern kann, so wie das keybase anbietet. 

- 6. Ja das Aushändigen der Passphrasen ist erstmal fehlgeschlagen. Das heißt aber nicht das man es nicht den Leuten die damit klar kommen nicht später noch anbieten kann. Insbesondere wenn Gradido vielleicht etwas bekannter ist und die Menschen Lust und Laune haben sich selbst mehr mit den Sicherheitsaspekten zu beschäftigen.